restaurant-menu/src/containers/Menu/Cart/Cart.js

```diff
@@ -0,0 +1,69 @@
+import './Cart.css';
+import React, {useState, useContext} from 'react';
+import OrderContext from '../../../OrderContext';
+import DishInCart from './DishInCart/DishInCart';
+import { useMediaQuery } from 'react-responsive'
+
+const Cart = props => {
+
+    const order = useContext(OrderContext)[0];
+    const delivery_costs = useContext(OrderContext)[2];
+    const isMobile = useMediaQuery({ query: '(max-width: 700px)' });
```

Comment:
Nice use of hooks. About the context - better to have an object with keys of `order` and `delivery_costs` (bad underscore naming, should be camel case), instead of hard coding integer indexes.